int32 number1
int32 number2